src/RL/utils/geometricflux/graphconv.jl

```diff
@@ -43,28 +43,34 @@ end
 function (g::GraphConv{<:AbstractMatrix,<:Any,meanPooling})(fgs::BatchedFeaturedGraph{Float32})
     A, X = fgs.graph, fgs.nf
 
+    sum = nothing
     Zygote.ignore() do
-        A = A ./ reshape(mapslices(x -> sum(eachrow(x)), A, dims=[1, 2]), 1, :, size(A, 3))
-
-        return BatchedFeaturedGraph{Float32}(
-            fgs.graph;
-            nf=g.σ.(g.weight1 ⊠ X .+ g.weight2 ⊠ X ⊠ A .+ g.bias),
-            ef=fgs.ef,
-            gf=fgs.gf
-        )
+        sum = replace(reshape(mapslices(x -> sum(eachrow(x)), A, dims=[1, 2]), 1, :, size(A, 3)), 0=>1)    
```

Comment:
What is the reason for adding a `replace( M, 0=>1)` ?

Reply:
edit : This is to avoid any division by 0, in case a node has no neighbors, which can happen for value that belongs to no domain after pruning.

Reply:
In the case where a node has no neighbor, the sum is zero, which will then cause problems in the division.

Putting a 1 in the division should not be a problem, because it will divide a vector composed only of 0. Any constant other than 1 would have done the job as well.